Automattic/vip-go-mu-plugins
# Display 'Not Proxied' flag when not proxied.

It can be hard to see if one is really considered to be proxied when accessing `wp-admin`, especially when sites are set up with a frontend caching service. Using such services will cause the calling user being considered not proxied by the backend servers, even if the caller is using the A8C proxy.
 
This mu-plugin will make `wp-admin` display a small banner when not proxied and logged in as Automattican, indicating that the caller is not proxied.

## Checklist


## Steps to Test

1. Log in to `wp-admin` on a site as an Automattician
2. Make sure you are not using an A8C proxy.
3. Notice that small flag saying that you are not proxied in the left hand corner.
